### ISSUE TITLE
loss of acquired purposeful hand skills

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11608]
**Preferred term label:**
loss of acquired purposeful hand skills

As in Rett syndrome

**Comment by pnrobinson** [comment]
also Postnatal deceleration of headgrowth 
Loss of acquired spoken language
Curate:  Clinical and functional outcomes in pediatric patients with Rett syndrome: a 15-year retrospective study

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001482
- label: Loss of acquired purposeful hand skills
- parents: [Motor regression (HP:0033044)](https://hpo.jax.org/app/browse/term/HP:0033044)
- pmid: PMID:21154482
- def: A type of neurodevelopmental regression characterized by deterioration or disappearance of previously acquired voluntary, goal-directed hand movements, such as reaching, grasping, manipulating objects, or self-feeding.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11496
- synonyms: Loss of purposeful hand skills(NONE)|Loss of purposeful hand movements(NONE)|Loss of acquired hand function(NONE)|Loss of purposeful hand use(NONE)|Regression of purposeful hand use(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001483
- label: Loss of acquired spoken language
- parents: [Cognitive regression (HP:0034332)](https://hpo.jax.org/app/browse/term/HP:0034332)
- pmid: PMID:21154482
- def: A type of neurodevelopmental regression characterized by deterioration or disappearance of previously acquired expressive verbal communication, including words, phrases, or sentences the individual had used meaningfully, after these skills had been acquired during development.
- comment: Loss of acquired language is based on best acquired spoken language skill, not strictly on the acquisition of distinct words or higher language skills.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11496
- synonyms: Loss of expressive language(NONE)|Regression of spoken language(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001484
- label: Postnatal deceleration of headgrowth
- parents: [Abnormality of skull size (HP:0000240)](https://hpo.jax.org/app/browse/term/HP:0000240)
- pmid: PMID:21154482
- def: Slowing of the rate of head circumference growth occurring after birth, in an individual with normal head circumference at birth, leading to progressive deviation from normal growth curves and, in many cases, acquired (postnatal, secondary) microcephaly.
- comment: The human brain undergoes dramatic changes in size and connectivity after birth. From birth to age six years, the brain increases in size by four-fold, reaching 90 percent of adult volume. During normal development, the growth of head circumference is largely driven by growth of underlying brain tissue.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11496
- synonyms:
